webapp/channels/src/components/invitation_modal/invitation_modal.tsx

```diff
@@ -206,20 +215,14 @@ export class InvitationModal extends React.PureComponent<Props, State> {
         if (this.state.invite.usersEmailsSearch !== '') {
             invites.notSent.push({
                 text: this.state.invite.usersEmailsSearch,
-                reason: this.props.intl.formatMessage({
-                    id: 'invitation-modal.confirm.not-valid-user-or-email',
-                    defaultMessage: 'Does not match a valid user or email.',
-                }),
+                reason: messages.notValidUserOrEmail,
```

Comment:
Changing `reason` to a `MessageDescriptor` also let me get rid of `injectIntl` here